diff --git a/dist/shared/vite-react-ssg.DUkzxBEb.mjs b/dist/shared/vite-react-ssg.DUkzxBEb.mjs
index 1eaadb929cce602514086e545d083438984e3f81..d0495f2a2ad613800284158bed794dd58c5a6746 100644
--- a/dist/shared/vite-react-ssg.DUkzxBEb.mjs
+++ b/dist/shared/vite-react-ssg.DUkzxBEb.mjs
@@ -457,7 +457,7 @@ class RemixAdapter {
     const styleCollector = getStyleCollector ? await getStyleCollector() : null;
     const helmetContext = {};
     let routerContext = null;
-    const { StaticRouterProvider, createStaticHandler, createStaticRouter } = await import('react-router-dom/server.js');
+    const { StaticRouterProvider, createStaticHandler, createStaticRouter } = await import('react-router');
     const { dataRoutes, query } = createStaticHandler([...routes], { basename: base });
     const _context = await query(request);
     if (_context instanceof Response)
